Demo/src/main/java/com/braintreepayments/demo/ShopperInsightsFragment.kt

```diff
@@ -31,142 +53,224 @@ class ShopperInsightsFragment : BaseFragment() {
     private lateinit var payPalClient: PayPalClient
     private lateinit var venmoClient: VenmoClient
 
+    private val venmoLauncher: VenmoLauncher = VenmoLauncher()
+    private val paypalLauncher: PayPalLauncher = PayPalLauncher()
+
+    private lateinit var venmoPendingRequest: VenmoPendingRequest
+    private lateinit var paypalPendingRequest: PayPalPendingRequest
+
     // TODO: Refactor Shopper Insights to remove BraintreeClient
-//    override fun onCreateView(
-//        inflater: LayoutInflater,
-//        container: ViewGroup?,
-//        savedInstanceState: Bundle?
-//    ): View? {
-//        braintreeClient = getBraintreeClient()
-//        shopperInsightsClient = ShopperInsightsClient(braintreeClient)
-//
-//        venmoClient = VenmoClient(this, braintreeClient)
-//        venmoClient.setListener(this)
-//
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        braintreeClient = BraintreeClient(requireContext(), authStringArg)
+        shopperInsightsClient = ShopperInsightsClient(braintreeClient)
+
+        venmoClient = VenmoClient(requireContext(), super.getAuthStringArg())
+
+        // TODO: What does manual browser switch do?
```

Comment:
Manual browser switching was a pattern added to v4 for use cases when encapsulating browser switching didn't work for merchant architecture - in v5 all of browser switching is "manual" (the merchants have to handle getting results) so separating the patterns isn't needed